frontend/src/libs/components/stepper/project-name-step/project-name-step.tsx

```diff
@@ -0,0 +1,49 @@
+import { Input } from '~/libs/components/input/input.js';
+import { useAppForm, useCallback } from '~/libs/hooks/hooks.js';
+import { siteNameValidationSchema } from '~/packages/sites/sites.js';
+import { type SiteNameDto } from '~/packages/sites/sites.js';
```

Comment:
since the path is the same, you can combine the call of imports

`import {
  type SiteNameDto,
  siteNameValidationSchema,
} from '~/packages/sites/sites.js';
`